docs/proposals/multi-version.md

```diff
@@ -0,0 +1,128 @@
+# Multi-Version-Landscaper-Instances
+
+This document contains a proposal how to support multiple versions of Landscaper Instances in one Landscaper as a 
+Service (LaaS) landscape.
+
+## Motivation
+
+A LaaS landscape should support different versions of a Landscaper instance (LI), i.e. different LandscaperDeployments
+install different versions of a [this](../../.landscaper/landscaper-instance) component. A LaaS landscape should support 
+a predefined set of LI versions, which change from release to release (probably if the LaaS component version changes)
+and a customer could select when to upgrade which of his instances.
+
+A LI is a complex software artefact used by the customers for their deployments. Upgrading a LI might include complex
+things like updating the used Helm version, the kubernetes version of the Landscaper resource shoot cluster and many
+more. Due to this complexity, it is impossible to ensure that LI updates might not harm an existing customer deployments.
+
+To reduce this risk, customers must have possibility to test a new LI component version with their development setup 
+first, before they decide to upgrade the LI component of their productive environment.
+
+## Requirements
+
+This chapter defines the rules for supported LI versions in a LaaS landscape:
+
+- Versioning:
+  - Patch: A new patch version of a LI should only include bugfixes and security fixes. It should not break a customer
+    setup.
+  - Minor: A new minor version of a LI could contain additional features, minor updates of the most important components,
+    like Helm or the Garden shoot cluster. Due to its complexity a minor update might break a customer setup. 
+  - Major: Groundbreaking changes. Might break a customer setup.
+
+- Supported LI version of a LaaS landscape:
+  - The list of supported versions of a LaaS landscape only contains one patch version (usually the latest) for every 
+    supported minor version. 
+
+- Basic upgrading rules
+  - A LI instance could be upgraded to the currently supported patch level of its current minor version. 
+  - If a LI instance runs on a currently supported patch level or its current minor version is not supported anymore,
+    then: 
+    - it could be upgraded to the next supported minor version or 
+    - if there is no supported higher minor version it could be upgraded to the lowest supported minor version of the 
+      next major version.
+
+- An upgrade of a LaaS landscape updates the list of supported LI versions according to the following rules:
+  - A supported minor version is set on deprecated, if there is a supported version with a higher major version or
+    a supported version with the same major but a higher minor version.
+  - A minor version could only be removed if it was deprecated for more than 3 months.
```

Comment:
```suggestion
  - A supported minor version can only be removed if it was deprecated for more than 3 months.
```

Reply:
Shall we specify the deprecation time in a dedicated sentence and replace it in this sentence with `A minor version can only be removed if the deprecation duration has ended.`?
When we decide to change the deprecation duration, we dont have to search and replace in the text.
What do you think?

Reply:
I have changed the wording and introduced deprecation duration.